### DISCUSSION TITLE
Version 1.2 Release Candidate 2

### DISCUSSION THREAD

**Discussion by ErinaSugino** [ErinaSugino/Starbound-SBR-Mod#96] [Update Builds]
## Version 1.2 Patchnotes
- Persistent crewmember data
- Player arousal effects and "in heat" mechanics
- NPC sub-genders
- Actor Relayering (with additional layers for ears and tails and offsets per species)
- Complete rework of sextalk
- Fixes for common issues like Anom's Outpost Overhaul, Vulpes Race, Cutebound Avians and forever-kids

## RC2 Notes
- Actually added player arousal
- Made baby color genetics once again more robust
- Added alternative body-anchored ears layer with option in species file
- Fixed layering with sextoys
- Added sextoy sextalk and fixed position setup for them
- Exhaustive testing needed before release.
- Known missing things from this build: Sextalk for cowgirl; potentially more efficient sextalk allocation.

### Important Notes
- Due to the complete overhaul, every sextalk altering mod is incompatible now.
- This build isn't tested

<hr /><em>This discussion was created from the release <a href='https://github.com/ErinaSugino/Starbound-SBR-Mod/releases/tag/v1.2_rc2'>Version 1.2 Release Candidate 2</a>.</em>

**Comment by luciodark** [comment]
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15104454/starbound.log)
There might be a few things wrong here. Half of my crew is invisible/stationary, nodes don't exist.

**Reply by ErinaSugino** [reply]
Your log says you messed up the main sexbound.config file, which would explain why everything breaks.
I've just booted the game with the exact files in this build and don't have this error - only a syntax error in one position file for the milkingmachine.

**Reply by luciodark** [reply]
Yeah, you're right. Idk where I broke it but replacing the files restored regular functionality. Arousal/heat still doesn't seem to work though.
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15125542/starbound.log)

**Comment by 777joshua7771** [comment]
Having a try of the 1.2 RC2 and curious on how to get the heat system to trigger, or how to make sure that it can trigger?

**Comment by ErinaSugino** [comment]
RC2 has a new .zip uploaded now.
Which fixes syntax errors breaking every Lustbound toy position, and adds yet another thing missing from player arousal... Which is the arousal resource. Third time's the charm.

**Reply by luciodark** [reply]
Buffs/debuffs for both arousal and heat seem to function. Is there an explanation of the exact intended effects somewhere?

**Reply by ErinaSugino** [reply]
> At 50%+ a light arousal effect is displayed.
Cosmetic only.
> At 90%+ a strong arousal effect is applied, reducing strength by 10%, increasing fertility by 10% and randomly reducing movement speed by 10% for 10 seconds.
> For species defined to have "heat cycles", applies a special status effect every 10-30 in-game days. This buff is like the strong arousal effect but also tries to play the species' defined moan effect at random and can only be removed by waiting a long while or becoming pregnant.
> Heat with pill: A weaker version of the heat status effect applied at half the remaining duration - 5% strength and random speed decrease, no fertility bonus, still moans.

**Reply by luciodark** [reply]
I see. It seems that none of the heat-related effects are occurring (aside from the status icon itself appearing and disappearing at the correct times) and there's nothing in the log regarding them.
